python_modules/libraries/dagster-celery-k8s/dagster_celery_k8s/launcher.py

```diff
@@ -337,22 +337,27 @@ def _get_validated_celery_k8s_executor_config(run_config):
     check.dict_param(run_config, "run_config")
 
     executor_config = run_config.get("execution", {})
+    execution_config_schema = resolve_to_config_type(celery_k8s_config())
+
+    # In run config on jobs, we don't have an executor key
```

Comment:
I guess this may make the error experience worse for legacy users, but since we can't tell from here whether we have a job i don't think it can be helped.